boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -34,6 +34,13 @@
     #define BOOTUTIL_CRYPTO_ECDSA_P256_HASH_SIZE (4 * 8)
 #endif
 
+/* Extract a member of the mbedtls context structure. */
+#if MBEDTLS_VERSION_NUMBER >= 0x03000000
+#define MBEDTLS_CONTEXT_MEMBER(X) MBEDTLS_PRIVATE(X)
+#else
+#define MBEDTLS_CONTEXT_MEMBER(X) X
+#endif
```

Comment:
Shouldn't this be moved to a common header - it is multiplicated in many files.

Reply:
I agree with you. Currently the relatively best place where we could define it is the `bootutil_priv.h` file, but it would inappropriate there too. I would suggest to simply create a new header, for example: `include/bootutil/crypto/common.h`.

Reply:
Done